packages/app-bridge/src/react/useBlockTemplates.ts

```diff
@@ -60,19 +77,29 @@ export const useBlockTemplates = (appBridge: AppBridgeBlock) => {
             await appBridge.deleteTemplateIdsFromBlockTemplateKey(key, oldTemplateIds);
             await appBridge.addTemplateIdsToBlockTemplateKey(key, newTemplateIds);
         } catch (error) {
-            console.error(error);
+            setErrorMessage(error as string);
```

Comment:
no `as` allowed (same for the rest of the `as string`, make sure to typecheck first 
```ts
if (typeof value === 'string') {
  setErrorMessage(error)
} else {
  setErrorMessage(calculateAString)
}
```

Reply:
Got it. Added the typechecking approach.